project-generator/src/main/java/com/redhat/ipaas/project/converter/DefaultProjectGenerator.java

```diff
@@ -47,6 +45,7 @@
 
     private final static ObjectMapper YAML_OBJECT_MAPPER = YamlHelper.createYamlMapper();
     private final static String PLACEHOLDER_FORMAT = "{{%s}}";
+    private final static Charset UTF8 = Charset.forName("UTF-8");
```

Comment:
ok.  I do wish the JDK had this as a constant somewhere since all JDKs are required to support this encoding.